### DISCUSSION TITLE
Hello, may I know how to integrate with object storage？

### DISCUSSION THREAD

**Discussion by feassh** [navidrome/navidrome#2660] [Q&A]
Hello, may I know how to integrate with object storage such as Minio? I have stored my songs in a self-built Minio instance, and I would like to use ND to connect to Minio's API to retrieve song files instead of directly using the local disk. Is it possible to achieve this?

**Comment by metalheim** [comment]
Not directly, as Navidrome only supports local filesystem https://github.com/navidrome/navidrome/issues/832

You can [rclone mount](https://rclone.org/commands/rclone_mount/) your object storage to a local filesystem. This way you can expose your object storage as a local filesystem to Navidrome.

Just beware that remote storage will always be slower to scan, not just because of network speed, but also API overhead.

**Reply by feassh** [reply]
Ok then, thanks for your reply!🙏
